lib/files/file_manager.ts

```diff
@@ -197,17 +196,17 @@ export class FileManager {
                 });
           }
         }
-      } else {
-        // We have not downloaded it before, or the version does not exist. Use the default content
-        // length of zero and download the file.
-        Downloader
-            .getFile(
-                binary, fileUrl, fileName, outputDir, contentLength, opt_proxy, opt_ignoreSSL,
-                callback)
-            .then(downloaded => {
-              resolve(downloaded);
-            });
       }
+      // We have not downloaded it before, or the version does not exist. Use the default content
+      // length of zero and download the file.
+      Downloader
```

Comment:
It looks like the main change here is that you moved this outside an `else` block, presumably in case you never find a binary with a matching version.  That makes sense, but then you should at a `return;` after the other `Downloader.getFile` so you don't end up downloading the file twice.